### PR TITLE
Add provisioning route for getting misc bridge info

### PR DESCRIPTION
**Pull request by turt2live**
Currently only the relay bot's username is exposed here.

### CODE REVIEW

File: mautrix_telegram/web/provisioning/spec.yaml [hunk 1]
```diff
@@ -24,6 +24,20 @@ tags:
...
 paths:
+  /bridge:
+    get:
+      operationId: get_bridge
+      summary: Get the bridge's information
+      tags: [Misc]
+      responses:
+        200:
+          description: The bridge information
+          schama:
```
Comment (tulir) [inline mautrix_telegram/web/provisioning/spec.yaml]: typo

File: mautrix_telegram/web/provisioning/spec.yaml [hunk 2]
```diff
@@ -24,6 +24,20 @@ tags:
 - name: Bridging
 
 paths:
+  /bridge:
+    get:
+      operationId: get_bridge
+      summary: Get the bridge's information
+      tags: [Misc]
```
Comment (tulir) [inline mautrix_telegram/web/provisioning/spec.yaml]: The "Misc" tag should be added to the tags array above paths

File: mautrix_telegram/web/provisioning/spec.yaml [hunk 3]
```diff
@@ -24,6 +24,20 @@ tags:
...
+      responses:
+        200:
+          description: The bridge information
+          schama:
+            type: object
+            properties:
+              relay_username:
```
Comment (tulir) [inline mautrix_telegram/web/provisioning/spec.yaml]: Spec doesn't match implementation

File: mautrix_telegram/web/provisioning/__init__.py [hunk 4]
```diff
@@ -64,6 +64,8 @@ def __init__(self, context: "Context") -> None:
         self.app.router.add_route("POST", f"{user_prefix}/login/send_code", self.send_code)
         self.app.router.add_route("POST", f"{user_prefix}/login/send_password", self.send_password)
 
+        self.app.router.add_route("GET", f"/bridge", self.bridge_info)
```
Comment (tulir) [inline mautrix_telegram/web/provisioning/__init__.py]: `"/bridge"` doesn't need `f`ormatting

File: mautrix_telegram/web/provisioning/spec.yaml (hunk 2 above)
Comment (turt2live) [reply mautrix_telegram/web/provisioning/spec.yaml]: Added

**Comment by turt2live** [comment]
@tulir this should be ready for another look. Apologies for the delay.

**Pull request merged**